static/index.html

```diff
@@ -95,6 +96,23 @@ <h3>Optional encryption key</h3>(encrypt your message before sending it to the
     </div>
 </body>
 
+<div class="footer">
+    <div>
+        <p>ABOUT TMPNOTES</p>
+    </div>
+    <div>
+        Thanks for using tmpnotes.com! TMPNOTES is an OpenSource project to allow notes to be passed simply and securely.
```

Comment:
```suggestion
        Thanks for using TMPNOTES! TMPNOTES is an open source project that allows notes to be passed simply and securely.
```
Small nits: 
* probably don't want to say `tmpnotes.com` here (unless we template with an env var) because this would be confusing for someone hosting it themselves using another domain :slightly_smiling_face: 
* reworded a bit, and I think `open source` is two words?